### PR TITLE
Improved a bit the docs website

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Changed the docs website theme

Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -18,6 +24,7 @@ Supported features of the specification:
 | AVRO Event Format             | :x: | :x: |
 | HTTP Protocol Binding         | :heavy_check_mark: | :heavy_check_mark: |
 | - [Vert.x](https://github.com/cloudevents/sdk-java/tree/master/http/vertx)        | :heavy_check_mark: | :heavy_check_mark: |
+| - [Jakarta Restful WS](https://github.com/cloudevents/sdk-java/tree/master/http/restful-ws) | :heavy_check_mark: | :heavy_check_mark: |
```
Comment (pierDipi) [inline docs/index.md:27]: `Vert.x` is preceded by a `-`, can you remove it?

File: docs/index.md [hunk 2]
```diff
@@ -27,23 +34,70 @@ Supported features of the specification:
...
 
+Event formats implementations are auto-discovered through an SPI. You can use them accessing through `EventFormatProvider`.
+For example, given you have in your classpath the [`cloudevents-json-jackson`] module, you can serialize/deserialize an event to/from JSON using:
+
+```java
+import io.cloudevents.core.format.EventFormatProvider;
+
+EventFormat format = EventFormatProvider
+  .getInstance()
+  .resolveFormat("application/cloudevents+json");
```
Comment (pierDipi) [inline docs/index.md:77]: Ditto

File: core/README.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ add `cloudevents-json-jackson` as a dependency and then:
 ```java
 import io.cloudevents.core.format.EventFormatProvider;
 
-EventFormat format = EventFormatProvider.getInstance().resolveFormat("application/json");
+EventFormat format = EventFormatProvider.getInstance().resolveFormat("application/cloudevents+json");
```
Comment (pierDipi) [inline core/README.md:45]: Can you use this constant? https://github.com/cloudevents/sdk-java/blob/12cca34d213be499464edebcc012efaafe069bfa/formats/json-jackson/src/main/java/io/cloudevents/jackson/JsonFormat.java#L31

File: docs/index.md (hunk 1 above)
Comment (slinkydeveloper) [reply docs/index.md:27]: That's intended :smile: 

image

Comment (pierDipi) [reply docs/index.md:27]: Oh, ok, that's fine :smile:

**Pull request merged**